README.md

```diff
@@ -47,11 +47,18 @@ What it sets up
 
 * [Bundler] for managing Ruby libraries
 * [Exuberant Ctags] for indexing files for vim tab completion
+* [Firefox]
 * [Foreman] for managing web processes
-* [hub] for interacting with the GitHub API
+* [Git] for version control
+* [GitX]
+* [Google Chrome]
+* [Google Drive]
+* [Google Hangouts]
 * [Heroku Toolbelt] for interacting with the Heroku API
 * [Homebrew] for managing operating system libraries
+* [hub] for interacting with the GitHub API
```

Comment:
It looks like thoughtbot didn't quite correctly alphabetize this in their README, so it's showing as moved in your commit. We should leave it in its current place to minimize future conflicts.